payment-circle/src/main/java/org/stellar/anchor/paymentservice/circle/CirclePaymentsService.java

```diff
@@ -268,14 +266,18 @@ private HttpClient getWebClient(boolean authenticated) {
      *                           format described in <a href="https://stellar.org/protocol/sep-38#asset-identification-format">SEP-38</a>.
      * @throws HttpException if the source account network is not CIRCLE.
      * @throws HttpException if the destination account network is not supported.
+     * @throws HttpException if the destination account is a bank and the idTag is not a valid email.
```

Comment:
Actually, Circle will receive it in the body as another parameter:
```js
{
  "source": {
    "type": "wallet",
    (...)
  },
  "destination": {
    "type": "wire",
    (...)
  },
  (...)
  "metadata": {
    "beneficiaryEmail": "satoshi@circle.com"
  }
}
```

Since the idea of the API is to be generic instead of mapping 1:1 with Circle's implementation, I've added it as a parameter of the destination account.

Also, `idTag` is meant to be a complementary identifier, so I think the email fits in that description:
https://github.com/stellar/java-stellar-anchor-sdk/blob/84130785915b811531abe15e3e11decd4575cfe5/core/src/main/java/org/stellar/anchor/paymentservice/Account.java#L15-L19

This is just the first iteration that can change in the future. Also, I intend to add some documentation with examples to make the API easier to use.